src/lib.rs

```diff
@@ -88,11 +105,23 @@ fn enumerate_project_files(language: &dyn SupportedLanguage) -> Vec<DirEntry> {
                 .build()
                 .unwrap(),
         )
-        .build()
-        .into_iter()
-        .filter_map(|entry| entry.ok())
-        .filter(|entry| entry.metadata().unwrap().is_file())
-        .collect()
+        .build_parallel()
```

Comment:
This gives you a [`WalkParallel`](https://docs.rs/ignore/latest/ignore/struct.WalkParallel.html) (instead of a `Walk`) which rather than giving you an iterator-like interface has you supply a callback to `.run()` that (returns a callback that) gets invoked for each enumerated project file